realscale.inx

```diff
@@ -16,16 +17,52 @@
     <item value="m">m</item>
     <item value="km">km</item>
   </param>
+  <param name="desc_unit" type="description">Large objects need a scale to be drawn on the page:</param>
+  <param name="showscale" type="boolean" _gui-text="Show Scale Ruler?">true</param>
+  <param name="drawnscale" type="enum" _gui-text="Drawing Scale:                    " _gui-description="-world unit for drawing">
```

Comment:
Why not allow the user to enter a value instead of selecting from this long list?